File: estructuraTablas/app/src/main/java/com/example/holamundo/MainActivity.java
package com.example.holamundo;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Log.i("Traza", "En evento Oncreate");
        TextView texto = new TextView(this);
        texto.setText("El ping pong me va a sacar de pobre");
        setContentView(texto);
        texto.setTextColor(getResources().getColor(R.color.yellow));

    }

    protected void salir(View view){
        finishActivity(0);
    }
}

File: miAplicacion/app/src/main/java/com/example/miaplicacion/MainActivity.kt
package com.example.miaplicacion

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

    }

    fun irACalculadora(view: View) {
        val intent = Intent(this, CalculatorActivity::class.java)
        startActivity(intent)
    }

    fun irATresRaya(view: View) {
        val intent = Intent(this, TresRayaActivity::class.java)
        startActivity(intent)
    }
}

